### DISCUSSION TITLE
Error of restarting a simulation on INL HPC

### DISCUSSION THREAD

**Discussion by xueyang94** [idaholab/moose#23047] [Q&A Modules: General] [answered]
Hello, I am trying to restart a simulation on INL HPC but got an error:

```
Using g280_out_cp/2770 for recovery.^[[39m
ERROR: Neither one of the following files can be located:
        'g280_out_cp/2770_mesh.cpr/1/header.cpr' nor
        'g280_out_cp/2770_mesh.cpr'
If you are running a parallel job, double check that you've created a split for 1 ranks.
Note: One of paths above may refer to a valid directory on your system, however we are attempting to read a valid header file.
```

Both of the paths mentioned are valid. I am running parallel, but not sure how to create a split for 1 ranks. In my PBS scheduler file I have `#PBS -l select=1:ncpus=24:mpiprocs=12`. Any suggestions?

**Comment by GiudGiud** [comment, marked as answer]
Hello

You booked 24 cpus / 12 mpi procs on the node, but did you use mpirun to run the job?
`mpirun -n 12 <executable)> -i ...`

The message is saying 1 rank which is odd if you are running in parallel. 
What is the header of the simulation reporting for the number of processes?

Guillaume

**Reply by xueyang94** [reply]
I have the keyword `mpirun`, but I did not specify number of processes `-n`. Now I doubt if it is actually mpi:

```
module load use.moose PETSc
mpirun /home/wuxuey/projects/moose/modules/phase_field/phase_field-opt -i /scratch/wuxuey/inputs/checkpoint_model4/g280.i --recover g280_out_cp/2770
```
If this is not parallel, why would I receive the error?

The header of the original simulation (the one I want to recover) says Num Processors is 12.

**Reply by GiudGiud** [reply]
You should specify -n 12.
What does the header of the new simulation say?

**Reply by GiudGiud** [reply]
You receive the error because the original split is for 12 procs, but you are trying to read with 1 process
